src/domain/myProfile/myProfileForm/MyProfileForm.tsx

```diff
@@ -162,4 +167,38 @@ const MyProfileForm: React.FC<Props> = ({
   );
 };
 
+const OrganisationsField: React.FC<{
+  name: string;
+  label: string;
+  helper: string;
+  placeholder: string;
+}> = ({ name, label, helper, placeholder }) => {
+  const { values, setFieldValue } = useFormikContext();
+  const { organisationProposals } = values as MyProfileFormFields;
```

Comment:
This could in one line with generic type:

```
const {
    values: { organisationProposals },
    setFieldValue,
  } = useFormikContext<MyProfileFormFields>();
```